src/pages/components/table-of-contents.mdx

```diff
@@ -42,14 +50,30 @@ The area above the Table of contents can be utilized in many ways—showing navi
 <Row>
 <Column colMd={8} colLg={8}>
 
-![Image of Table of contents component](../../images/component/table-of-contents/table-of-content-with-heading-content.jpg)
+![Image of With heading content variation](../../images/component/table-of-contents/table-of-content-with-heading-content.jpg)
+
+</Column>
+</Row>
+
+<Title>Use case</Title>
+
+![Image of With heading content variation usecase](../../images/component/table-of-contents/table-of-content-with-heading-content-use-case.jpg)
+
+
+### Horizontal
+The Horizontal variation displays pre-defined sections of the page and provides a convenient way to navigate those sections. It also sticks to the top of the page when scrolling for easy access.
+
+<Row>
+
+![Image of Horizontal variation](../../images/component/table-of-contents/table-of-contents-horizontal.jpg)
 
 </Column>
 </Row>
 
 <Title>Use case</Title>
 
-![Image of Table of contents component](../../images/component/table-of-contents/table-of-content-with-heading-content-use-case.jpg)
+![Image of Horizontal variation usecase](../../images/component/table-of-contents/table-of-contents-horizontal-use-case.jpg.)
+
```

Comment:
**When to use**
* Use the Table of contents – horizontal variation with section level components, where the section title occupies the first four columns of the 16 column grid and the Table of contents – vertical could conflict with the content.